front/helm/templates/job-page.yaml

```diff
@@ -120,8 +120,10 @@ spec:
               value: {{ .Values.gettingStarted.mountPath | quote }}
             - name: WORKFLOW_TEMPLATES_YAMLS_PATH
               value: {{ .Values.workflowTemplates.mountPath | quote }}
+{{- if eq .Values.global.edition "ce" }}
             - name: "SKIP_VELOCITY"
```

Comment:
One thing here: we have a lot of environment variables (SKIP_VELOCITY, HIDE_PROMOTIONS, CE_ROLES, ...) that are based included or not based on `.Values.global.edition`. It's probably simpler to just pass a single `EDITION` environment variable instead of all of those

Reply:
I mean replacing them with a single var. Yes, we lose configurability of the service, but configurability makes things more complicated. Those variables are only gonna be tweaked based on the edition, no?

Reply:
@lucaspin I would rather keep them like this at the moment, and reduce later down the line if we decide to do so